client/apps/edit/components/section_text/index.coffee

```diff
@@ -1,145 +1,397 @@
+    urlValue: null
+
   componentWillMount: ->
     @channel = new Channel sd.CURRENT_CHANNEL
 
   componentDidMount: ->
-    @attachScribe()
-    @componentDidUpdate()
+    if @props.section.get('body')?.length
+      blocksFromHTML = @convertFromHTML(@props.section.get('body'))
+      @setState
+        html: @props.section.get('body')
+        editorState: EditorState.createWithContent(blocksFromHTML, new CompositeDecorator(decorators))
 
-  componentDidUpdate: ->
-    $(@refs.editable).focus() if @props.editing
+  componentWillReceiveProps: (nextProps) ->
+    if @props.editing and !nextProps.editing
+      @setState focus: false, showUrlInput: false, urlValue: null
+    else
+      @focus() if @props.editing
 
-  shouldComponentUpdate: (nextProps) ->
-    not (nextProps.editing and @props.editing)
+  onChange: (editorState) ->
+    html = @convertToHtml editorState
+    @setState editorState: editorState, html: html
+    @props.section.set('body', html)
 
   onClickOff: ->
-    @setBody()
     @props.section.destroy() if $(@props.section.get('body')).text() is ''
 
-  setBody: ->
-    @props.section.set body: $(@refs.editable).html()
-
-  attachScribe: ->
-    @scribe = new Scribe @refs.editable
-    @scribe.use scribePluginSanitizeGoogleDoc()
-    @scribe.use scribePluginSanitizer {
-      tags:
-        p: true
-        b: true
-        i: true
-        br: true
-        a: { href: true, target: '_blank', name: true, class: true, 'data-id': true}
-        h2: true
-        h3: true
-        ol: true
-        ul: true
-        li: true
+  focus: (e) ->
+    @setState focus: true
+    @refs.editor.focus()
+
+  onBlur: ->
+    @setState focus: false
+
+  convertFromHTML: (html) ->
+    blocksFromHTML = convertFromHTML({
+      htmlToStyle: (nodeName, node, currentStyle) ->
+        if nodeName is 'span' and node.style.textDecoration is 'line-through'
+          return currentStyle.add 'STRIKETHROUGH'
+        else
+          return currentStyle
+      htmlToEntity: (nodeName, node) ->
+        if nodeName is 'a'
+          data = {url: node.href, name: node.name, className: node.classList.toString()}
+          return Entity.create(
+              'LINK',
+              'MUTABLE',
+              data
+          )
+        if nodeName is 'p' and node.innerHTML is '<br>'
+          node.innerHTML = '' # remove <br>, it renders extra breaks in editor
+      })(html)
+    return blocksFromHTML
+
+  convertToHtml: (editorState) ->
+    html = convertToHTML({
+      entityToHTML: (entity, originalText) ->
+        if entity.type is 'LINK'
+          sanitizeName = originalText.split(' ')[0].replace(/[.,\/#!$%\^&\*;:{}=\-_`’'~()]/g,"")
+          name = if entity.data.name then ' name="' + sanitizeName + '"' else ''
+          if entity.data.className.includes('is-follow-link')
+            return '<a href="' + entity.data.url + '" class="' + entity.data.className + '"' + name + '>' + originalText + '</a><a class="entity-follow artist-follow"></a>'
```

Comment:
Sorry, what I mean is not `name` but `data-id` which should be present on the actually follow button, the second link. 

Here's a sample of how the follow links look like currently:
<a href=\"https://www.artsy.net/artist/ej-hill\" class=\"is-jump-link is-follow-link\" name=\"EJ_Hill\">EJ Hill</a><a data-id=\"ej-hill\" class=\"entity-follow artist-follow\"></a>

[and here's what I did in the Scribe plugin](https://github.com/artsy/scribe-plugin-follow-link-tooltip/blob/master/index.js#L193)